src/auto/functions.rs

```diff
@@ -47,9 +47,7 @@ pub fn assertion_message_cmpstr(domain: &str, file: &str, line: i32, func: &str,
     }
 }
 
-pub fn assertion_message_expr<'a, 'b, P: Into<Option<&'a str>>, Q: Into<Option<&'b str>>>(domain: P, file: &str, line: i32, func: &str, expr: Q) {
-    let domain = domain.into();
-    let expr = expr.into();
+pub fn assertion_message_expr(domain: Option<&str>, file: &str, line: i32, func: &str, expr: Option<&str>) {
```

Comment:
Why do we even have this function in the bindings? Someone should review all the global functions we have generated here...